app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -43,6 +56,13 @@ def validate_import_vm
     Gem::Version.new(api_version) >= Gem::Version.new('4')
   end
 
+  def submit_configure_imported_vm_networks(userid, vm_id)
```

Comment:
This `do_stuff` and `submit_do_stuff` is actually used for import vm `submit_import_vm` + `import_vm` and I believe was originally conceived together with @mkanoor in the initial v2v design phase. Here I'm just trying to maintain consistency